e2core/satbackend/orchestrator.go

```diff
@@ -86,22 +92,18 @@ func (o *Orchestrator) Shutdown() {
 	o.wg.Wait()
 }
 
-func (o *Orchestrator) reconcileConstellation(appSource appsource.AppSource) {
-	ovv, err := appSource.Overview()
-	if err != nil {
-		o.logger.Error(errors.Wrap(err, "failed to app.Overview"))
+func (o *Orchestrator) reconcileConstellation(syncer *syncer.Syncer) {
```

Comment:
this method does roughly the same as syncer.Run(). What's the difference between the two? Or rather could we use syncer.Run to do the thing here instead of doing mostly the same thing in reconcileConstellation?

Reply:
I think we can possibly merge them in the future, but for now this works ™️ so I'd call that a future refactor.